src/utils/UserLocalStorage.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * class for accessing local storage for user
+ */
+export class UserLocalStorage {
```

Comment:
We should probably aim to use functional programming with hooks, using the React hooks API will help maintain good app performance since they (useEffect, useState) are already optimized to be used in a react app.

> A custom Hook is a JavaScript function whose name starts with ”use” and that may call other Hooks.
https://reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook

Thus, I would suggest converting this class to a custom hook.

Reply:
@mannycolon - I already made one (useUserLocalStorage), but it did not work for the scripture version history because the same localStorage item is shared between all scripture versions.  It created some hazards since all cards were validating on app startup.  That's why I went stateless for scripture version history.  Will think about if there is another approach, but I dread doing a major refactor there.

Reply:
@mannycolon - I'm going to try moving the scripture version state into the app since it is share by all the scripture panes.  Maybe that will fix the hazards from having all scripture panes trying to access the same local storage.